### DISCUSSION TITLE
Support for Metabase in non dev mode

### DISCUSSION THREAD

**Discussion by arunaruljothi** [geldata/gel#5311] [Q&A] [answered]
Hello, 

I noticed that support for metabase was completed here: #4814 but it looks like postgres protocol support is only available in dev mode. Is this going to be released along with version 3.0 ? Are there any other work arounds for getting it to work on non-development versions of edgedb? 

Thanks

**Comment by elprans** [comment, marked as answer]
Support for Postgres emulation will be released in 3.0 (beta very soon).  There are no plans to backport it to 2.x.
